packages/react-instantsearch/src/connectors/connectHierarchicalMenu.test.js

```diff
@@ -169,6 +171,189 @@ describe('connectHierarchicalMenu', () => {
       expect(props.items).toEqual(['items']);
     });
 
+    it('is applying transformValue before applying transformItems', () => {
```

Comment:
Not sure that this is the best test title. What about `apply limit after transforming items`?